packages/website/package.json

```diff
@@ -47,6 +47,13 @@
 	},
 	"homepage": "https://discord.js.org",
 	"dependencies": {
+		"@emotion/react": "^11.10.0",
+		"@emotion/server": "^11.10.0",
+		"@mantine/core": "^5.1.5",
+		"@mantine/hooks": "^5.1.5",
+		"@mantine/next": "^5.1.5",
```

Comment:
Have you tried https://www.npmjs.com/package/nextjs-progressbar, it doesn't need any extra configurations. You just need to add `<NextNProgress />` in `_app.ts`.

Reply:
I selected the wrong line number. This suggestion was for `@mantine/nprogress`

Reply:
I'd rather stay in the same UI framework.